src/components/FoodBox.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+
+class FoodBox extends Component {
+  render() {
+
+    const {food} = this.props
+
+    return (
+      <div>
+
+        {
+            food.map((singleFood, i) => {
```

Comment:
This approach works good, but for readability and even logically, it would make more sense if you apply the map in the previous component (App.js) and for each iteration, you render the FoodBox component with the required props.